src/resolve/resolved-dns-answer.c

```diff
@@ -152,20 +154,22 @@ static int dns_answer_add_raw_all(DnsAnswer *a, DnsAnswer *source) {
                                 item->rr,
                                 item->ifindex,
                                 item->flags,
-                                item->rrsig);
+                                item->rrsig,
+                                USEC_INFINITY);
```

Comment:
please prefix with `/* until= */`

(we do that these days in func calls where the passed parameter gives no hint what it is)

Reply:
Where? It's not visible here, so maybe it ended up in some other commit.